bioanalyzer_service/README.md

```diff
@@ -0,0 +1,103 @@
+# Bioanalyzer Service
+
+Spring Boot microservice that abstracts common analysis we wish to run on bio images. An example is to quality check an 
+image of a fingerprint.
+
+
+## Getting Started
+
+These instructions will get you a copy of the project up and running on your local machine for development and testing
+purposes. See deployment for notes on how to deploy the project on a live system.
+
+### Prerequisites
+
+For building and running the application you need:
+
+- JDK 1.8
+- Gradle
+- Git
+- Docker
+
+### Installing
+
+Should always be built with a docker container given dependency on NFIQ2 which is an absolute pain to set up. Note that
```

Comment:
small semantic thing:  I would prefer to see this written in 3rd person (aka no 'we') and sentences starting with nouns etc...I am not an english major so maybe take this suggestion with a grain of salt.

Reply:
I was trying not to change too much from the original, but there's no harm in making some tweaks here. :)